ui/ui-components/components/export/List.tsx

```diff
@@ -274,26 +278,26 @@ export default function ExportsList(props) {
   );
 }
 
-ExportsList.hydrate = async (appContext: NextPageContext) => {
-  const client = generateClient(appContext);
+ExportsList.hydrate = async (ctx: GetServerSidePropsContext) => {
+  const client = generateClient(ctx);
 
-  const { id, limit, offset, state, recordId, destinationId } =
-    appContext.query;
-  const { groups } = await client.request("get", `/groups`);
+  const { id, limit, offset, state, recordId, destinationId } = ctx.query;
+  const { groups } = await client.request<Actions.GroupsList>("get", `/groups`);
 
   let exportProcessorId: string;
```

Comment:
Nitpick, though I know this was around before:
```suggestion
  let exportProcessorId = "";
```

Reply:
Will leave as is because it was like that before. I'm not sure if this will cause the action to fail later on, and I'd like not to specifically test this.